### DISCUSSION TITLE
Postal not sending bounce messages to return path

### DISCUSSION THREAD

**Discussion by chrisdeeming** [postalserver/postal#2746] [Help with using Postal] [answered]
I've got a bunch of test email addresses that will reliably bounce. If I send an email to one of these addresses through Postal, the emails get marked by Postal as "Hard Fail", but no notification of the bounce is sent to the return path. Am I missing something here? This kind of thing was working fine before we migrated from our existing postfix setup.

<img width="1154" alt="image" src="https://github.com/postalserver/postal/assets/2158514/a12a040f-9e96-4ae9-a263-d51090808f13">

**Comment by chrisdeeming** [comment]
For clarity, I am familiar with the documentation, albeit slightly confused.

The return path I am including in outgoing emails actually point to our domain hosted at Gmail. We don't want or need Postal to handle any sort of incoming email.

But if an email bounces or hard fails we just want that sent by email back to the return path.

**Reply by willpower232** [reply, marked as answer]
As the other email server did not accept the email, there is no return path email message to be handled.

You would need to set up webhooks to be notified of hard fails.

**Reply by chrisdeeming** [reply]
This is the conclusion I have come to. But, why? Why is Postal unable to generate an email on hard fail and send it to the return path? This is what Postfix does. It's not insurmountable but it seems silly to me that an email server can't just...send an email. I'm not the first to think this, looking at some of the unanswered questions here, and I don't think I will be the last. People already have infrastructure or services set up to automatically process these kinds of failed emails so it's a shame it isn't an option.
